Makefile

```diff
@@ -11,14 +11,18 @@ EMBEDDED_OPERATOR_CHART_NAME = embedded-cluster-operator
 EMBEDDED_OPERATOR_CHART_VERSION = 0.36.5
 EMBEDDED_OPERATOR_UTILS_IMAGE = busybox:1.36.1
 EMBEDDED_CLUSTER_OPERATOR_IMAGE_OVERRIDE =
-OPENEBS_CHART_VERSION = 4.1.0
-OPENEBS_UTILS_VERSION = 4.1.0
+OPENEBS_CHART_VERSION = 4.0.1
```

Comment:
Somehow this seems to be downgrading OpenEBS. This does not feel right, maybe this is from before `4.1.0` became available.